test/unit/mdc-floating-label/mdc-floating-label.test.js

```diff
@@ -35,6 +36,27 @@ function setupTest() {
   return {root, component};
 }
 
+test('#shake calls the foundation shake method', () => {
+  const {component} = setupTest();
+  component.foundation_.shake = td.func();
+  component.shake(true);
+  td.verify(component.shake(true), {times: 1});
```

Comment:
This is literally verifying the thing you just called. I think this is supposed to be foundation?

Reply:
Ah! Yes...I'm supposed to be testing the foundation like in the subsequent tests. Thanks for catching.